parser.py

```diff
@@ -216,14 +215,14 @@ def convert_tweet(tweet, username, archive_media_folder, output_media_folder_nam
                     html += f'<img src="{new_url}"/>'
                     # Save the online location of the best-quality version of this file, for later upgrading if wanted
                     best_quality_url = f'https://pbs.twimg.com/media/{original_filename}:orig'
-                    media_sources.append((os.path.join(output_media_folder_name, archive_media_filename), best_quality_url))
+                    media_sources.append((os.path.join(paths.dir_output_media, archive_media_filename), best_quality_url))
                 else:
                     # Is there any other file that includes the tweet_id in its filename?
-                    archive_media_paths = glob.glob(os.path.join(archive_media_folder, tweet_id_str + '*'))
+                    archive_media_paths = glob.glob(os.path.join(paths.dir_input_media, tweet_id_str + '*'))
                     if len(archive_media_paths) > 0:
                         for archive_media_path in archive_media_paths:
                             archive_media_filename = os.path.split(archive_media_path)[-1]
-                            media_url = f'{output_media_folder_name}{archive_media_filename}'
+                            media_url = f'{paths.dir_output_media}{archive_media_filename}'
```

Comment:
Sadly this approach doesn't work any more. All the media files get copied into `./media*.*` instead of `./media/*.*`

Reply:
I think we need to do: (untested)

```
file_output_media = os.path.join(paths.dir_output_media, archive_media_filename)
media_url = f'{os.path.split(paths.dir_output_media)[1]}/{archive_media_filename}'
```
and make dir_output a class member of PathConfig.

Reply:
This patch seems to work for me but test it on your end:

https://gist.github.com/timhutton/860c39695de8aeefda5a7c24187b826a

Reply:
Looks good to me and works for me. So I'll commit that patch...

The computation of the relative URL seems a bit quirky to me, since it assumes a specific directory configuration. But so did the old version. I guess the proper way would be to call [`os.path.relpath`](https://docs.python.org/3/library/os.path.html#os.path.relpath) on the two absolute paths of the `html` or `md` file and the media file. This would not break when we change the directory layout. But would return a path with backslashes on Windows, and I guess even there a `file://`-URL should use forward slashes...

So, I'd be fine to keep it as is for now, and introduce that relpath-stuff during #99.